packages/generator/src/sdk-metadata/__snapshots__/sdk-metadata.spec.ts.snap

```diff
@@ -2,16 +2,23 @@
 
 exports[`pregenerated-lib returns description of the service 1`] = `"SAP Cloud SDK for JavaScript: Virtual Data Model (VDM) for service test-service"`;
 
-exports[`sdk-metadata [E2E] generates the JS metadata content for services with pregenerated lib 1`] = `
+exports[`sdk-metadata generates the File names 1`] = `
+Object {
+  "clientFileName": "MyService_CLIENT_JS.json",
+  "headerFileName": "MyService_HEADER.json",
+}
+`;
+
+exports[`sdk-metadata generates the JS metadata content for services with pregenerated lib 1`] = `
 Object {
   "emergencyObject": undefined,
   "generationAndUsage": Object {
     "apiSpecificUsage": Object {
-      "instructions": "import { TestEntity } from '@sap/cloud-sdk-vdm-business-partner-service';
+      "instructions": "import { TestEntity } from 'non-existing-package';
```

Comment:
Is this intended? Seems odd.

Reply:
I did not really pay attention. Since it does not matter what is used there I put the test service.